### ISSUE TITLE
Allow "paste linked" - linking notes (and others) between independent staves, or to different places in one part.

### ISSUE THREAD

**Issue by sammik** [musescore/MuseScore#20918]
### Your idea

It may be useful, to have ability to link several pieces of melodies between independent parts, if several instruments play unisono passages (or same melody is repeated in ine part multiple times).

### Problem to be solved

In unisono passages, every change needs to be made to all unisono parts.
(It is easy, to make error - forget some change in some part)

### Additional context

It may be done by adding "Paste linked command".

Theese linked notes may have different (violet) colour, when selected (one selected -> all linked coloured).

Note may be unlinked by clicking "unlink" in context menu. 
(In Properties tab, if linked, there may appear ticker "linked", which once note became unlinked disappears.)

**Comment by oMrSmith** [comment]
This might also be useful for adding cue notes...

**Comment by sammik** [comment]
> This might also be useful for adding cue notes...

I think cue notes is bit different situation, because they have some differencies (at least cue size, and "play otion") and especially we dont want to remove "root" notes, when deleting cue notes.

**Comment by oMrSmith** [comment]
> we dont want to remove "root" notes, when deleting cue notes

That's true but it would be nice if cue notes would adapt changes in the part they are coming from - like it works in Dorico.

**Comment by oktophonie** [comment]
Proper cues are going to be their own feature, with all sorts of options for how they are shown. Obviously it's a form of linking.

Implementing this as a general solution purely for the purpose saving a bit of time for passages where instruments are sometimes playing the same thing seems a false economy to me, personally (and would be more trouble than it's worth).

**Comment by lgvr123** [comment]
Also really useful, for riff-based music (e.g. pop/rock/metal/... scores). In such scores, it is not uncommon to have the riffs described somewhere on the page, as "Riff1" and the few measures for that riff, later "Riff2" ibid., ...
And in the score itself having in the instrument staff the sole mention of "play riff 1", "play riff2".
Here is an example :
![subscore](https://github.com/user-attachments/assets/dcc9528e-b290-4dc4-95ae-bc607854c8f6)

To have it played correctly, I had to duplicate the contrabass instrument, paste the riff measures all over and hide the instrument in the end.
